Kickstarter-iOS/Views/Controllers/ProjectPageViewController.swift

```diff
@@ -0,0 +1,415 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+public final class ProjectPageViewController: UIViewController, MessageBannerViewControllerPresenting {
+  // MARK: Properties
+
+  private enum NavigationButtonSizes: CGFloat {
```

Comment:
It looks like the convention we use everywhere else in the app follows this:

`private enum Layout {
  enum Button {
    static let height: CGFloat = 34
  }
`